lib/models/application.coffee

```diff
@@ -85,21 +87,22 @@ getApplicationModel = (deps, opts) ->
 	exports.getAll = (options = {}, callback) ->
 		callback = findCallback(arguments)
 
-		token.getUserId().then (userId) ->
-			return pine.get
-				resource: 'application'
-				options:
-					mergePineOptions
-						orderby: 'app_name asc'
-						filter:
-							user: userId
-					, options
+		auth.getUserId()
+			.then (userId) ->
+				return pine.get
+					resource: 'application'
+					options:
+						mergePineOptions
+							orderby: 'app_name asc'
+							filter:
+								user: userId
+						, options
 
-		.map (application) ->
-			normalizeApplication(application)
-			return application
+			.map (application) ->
+				normalizeApplication(application)
+				return application
 
-		.asCallback(callback)
+			.asCallback(callback)
```

Comment:
Nit: I normally wouldn't indent the `then`/`map`/`asCallback` here - I'd leave them lining up with the start of `auth.getUserId()`